packages/go/analysis/go.mod

```diff
@@ -21,6 +21,8 @@ go 1.23
 require (
 	github.com/RoaringBitmap/roaring v1.9.4
 	github.com/bloodhoundad/azurehound/v2 v2.0.1
+	github.com/go-faker/faker/v4 v4.6.0
```

Comment:
I'd like to get more input on whether we want faker in the project. Ultimately, it's a somewhat useful library, but we haven't used anything like it before so I'd like to vet it's worth adding a dependency here